.gitignore

```diff
@@ -6,6 +6,10 @@ composer.lock
 .DS_Store
 Thumbs.db
 wandb/
+/ios/OpenBot/Pods/
```

Comment:
There should be another .gitignore file at the level `ios/OpenBot` that has `Pods`, etc. in it. Does not make sense to put this at the root level.